DTS-STN/secure-client-hub
# Add cypress tests for Login/Logout pages

## [ADO-128751](https://dev.azure.com/VP-BD/DECD/_workitems/edit/128751)

### Description
This PR adds simple cypress tests for the login and logout pages. As we cannot easily test our authentication flow through Cypress at this time due to the flow being behind firewall, these tests only test if the pages are rendering and tests that they properly redirect in the event that auth is disabled.

### What to test for/How to test
1. Pull in branch
2. Type `npm run e2e`
3. Run the tests for Login and Logout and confirm there are no errors
